policy/policy-authority.go

```diff
@@ -23,26 +29,31 @@ type AuthorityImpl struct {
 	log *blog.AuditLogger
 	DB  *AuthorityDatabaseImpl
 
-	EnforceWhitelist  bool
+	blacklist   map[string]bool
+	blacklistMu sync.RWMutex
+
 	enabledChallenges map[string]bool
 	pseudoRNG         *rand.Rand
 }
 
 // New constructs a Policy Authority.
-func New(dbMap *gorp.DbMap, enforceWhitelist bool, challengeTypes map[string]bool) (*AuthorityImpl, error) {
+func New(dbMap *gorp.DbMap, _ bool, challengeTypes map[string]bool) (*AuthorityImpl, error) {
```

Comment:
should add a TODO or something about rewriting the callers to remove the param later